diff --git a/node_modules/react-native-check-app-install/android/build.gradle b/node_modules/react-native-check-app-install/android/build.gradle
index 7caf7e1..db54782 100644
--- a/node_modules/react-native-check-app-install/android/build.gradle
+++ b/node_modules/react-native-check-app-install/android/build.gradle
@@ -11,8 +11,8 @@ buildscript {
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    compileSdkVersion 34
+    buildToolsVersion "34.0.0"
 
     defaultConfig {
         minSdkVersion 16
@@ -35,5 +35,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
 }
\ No newline at end of file
